docs/src/05-mgx-taxonomic-profiles.jmd

```diff
@@ -368,33 +376,30 @@ We can also annotate this plot with the age of the kids
 ```julia
 plot(
     abundanceplot(kids, srt = kids_hcl.order, title="Kids, top 10 species",
-        xticks=false, color=color4', legend=false),
+        xticks=false, color=color4', legend=false, linecolor=:match, linestyle=:dot),
     heatmap(collect(Float64[ismissing(x) ? 0 : x / 365 for x in focusmeta[:correctedAgeDays][kids_hcl.order]]'),
         xticks=false, yticks=false, color=:PuBu, colorbar=false),
     layout = grid(2,1,heights=[0.9, 0.1])
     )
 ```
```

Comment:
<img width="997" alt="Screen Shot 2019-07-08 at 12 10 02 PM" src="https://user-images.githubusercontent.com/30243213/60825353-853d1b80-a179-11e9-994a-482c04f4b1fe.png">
This plot doesn't have a legend for the age annotation.